.github/ISSUE_TEMPLATE/❓-question.md

```diff
@@ -0,0 +1,22 @@
+---
+name: "❓ Question"
+about: Ask a question about a feature, proposal, or if you just need help!
+title: "❓ Question"
+labels: question
+assignees: ''
+---
+
+## ❓ Question
```

Comment:
Is this a format we've ever used? What's the use-case?

(It seems like a template that's good for outside contributors, so I feel supportive of it! But if it's for the team, this seems like something we mostly use slack or daily scrum for, no? Should we be changing our behaviour to bring these into tickets?)

Reply:
Yup, we use it a lot :) Now, is a ticket the right place for it is a more philosophical topic! I think github discussions or slack threads are a more appropriate place for it. But for now, we are using it so I created a template to try to at least standardize it.

https://github.com/GCTC-NTGC/gc-digital-talent/labels/blocked%3A%20question

Reply:
thanks!

And ooooh that's neat that linking to an issue renders it in a comment. Didn't realize that :/

Reply:
ha! Neither did I until I did it :P